hermes-console-vue/src/views/topic/topic-header/TopicHeader.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+  import type { Owner } from '@/api/owner';
+  import type { TopicWithSchema } from '@/api/topic';
+
+  const props = defineProps<{
+    topic: TopicWithSchema;
+    owner: Owner;
+  }>();
+</script>
+
+<template>
+  <v-card density="compact">
+    <v-card-item>
+      <p class="text-overline">{{ $t('topicView.header.topic') }}</p>
+      <div class="d-flex justify-space-between">
+        <p class="text-h4 font-weight-bold">{{ props.topic.name }}</p>
+        <v-tooltip text="Copy event name">
```

Comment:
What do You think about moving it to i18n? Also, what about changing it to `Copy topic name`?